MeowdoptMeApp/src/SheltersScreen.tsx

```diff
@@ -1,24 +1,88 @@
 import * as React from 'react';
-import {View, Text, StyleSheet} from 'react-native';
+import {useState, useContext} from 'react';
+import {ScreenContext} from './Context';
+import {View, Text, StyleSheet, Pressable} from 'react-native';
 
 function SheltersScreen() {
+  const [page, setPage] = useState(0);   // 0 to sheltersList, 1 - mapa
```

Comment:
Making it a numerical value and then add a comment is not the best option here - code should be as self explanatory as possible. Consider making the state a string, for eg. `const [page, setPage] = useState('list')` or making it a boolean value: `const [isSheltersList, setIsSheltersList] = useState(true)`.